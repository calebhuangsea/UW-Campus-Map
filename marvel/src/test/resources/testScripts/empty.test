# Test find path with an empty graph

CreateGraph g

FindPath g n1 n2