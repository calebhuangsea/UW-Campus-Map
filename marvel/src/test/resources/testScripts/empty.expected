# Test find path with an empty graph

created graph g

unknown: n1
unknown: n2